integration-tests/container/benchmark/Makefile

```diff
@@ -7,9 +10,22 @@ COLLECTOR_QA_BENCHMARK_TAG=phoronix-$(COLLECTOR_QA_TAG)
 endif
 
 .PHONY: all
-all:
-	@docker build -t quay.io/rhacs-eng/collector-performance:$(COLLECTOR_QA_BENCHMARK_TAG) .
+all: build
+
+.PHONY: build
+build:
+	@docker buildx build --load --platform ${PLATFORM} \
+                -t quay.io/rhacs-eng/collector-performance:$(COLLECTOR_QA_BENCHMARK_TAG)-$(IMAGE_ARCH) .
+
+.PHONY: build-and-push
+build-and-push: build
+	@docker push quay.io/rhacs-eng/collector-performance:$(COLLECTOR_QA_BENCHMARK_TAG)-$(IMAGE_ARCH)
+
+.PHONY: manifest
+manifest:
+	@docker manifest create quay.io/rhacs-eng/collector-performance:$(COLLECTOR_QA_BENCHMARK_TAG) \
+		quay.io/rhacs-eng/collector-performance:$(COLLECTOR_QA_BENCHMARK_TAG)-amd64 \
+		quay.io/rhacs-eng/collector-performance:$(COLLECTOR_QA_BENCHMARK_TAG)-s390x \
+		quay.io/rhacs-eng/collector-performance:$(COLLECTOR_QA_BENCHMARK_TAG)-ppc64le
```

Comment:
[nit] not sure about this, but we could have a list of archs in `Makefile-constants.mk` and iterate over it. It would avoid changing the list everywhere upon addition. However, a change to the constants is not watched for modifications, and the current code is readable... so definitely not sure :smile: